### PR TITLE
Fix GH#19322: [MusicXML Import] Crash if stop wavy-line is missing

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #19322

### CODE REVIEW

File: src/engraving/playback/filters/spannerfilter.cpp [hunk 1]
```diff
@@ -53,7 +53,10 @@ bool SpannerFilter::isPlayable(const EngravingItem* item, const RenderingContext
 int SpannerFilter::spannerActualDurationTicks(const Spanner* spanner, const int nominalDurationTicks)
 {
     if (spanner->type() == ElementType::TRILL) {
-        return spanner->endSegment()->tick().ticks() - spanner->tick().ticks() - 1;
+        if (spanner->endSegment()) {
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/engraving/playback/filters/spannerfilter.cpp]: done, and rebased in the due course

**Review by RomanPudashkin — APPROVED** [review]

**Comment by cbjeukendrup** [comment]
Note that this code will be removed in #18766, which seems to make this PR obsolete

**Comment by Jojo-Schmitz** [comment]
So #18766 would need to get tested against #19322

**Comment by Jojo-Schmitz** [comment]
I just did, and there's no crash

**Pull request closed without merging**